### PR TITLE
Update sql-statement-show-profiles.md to reference statement summary tables

### PR DESCRIPTION
**Pull request by adamf-db**
### What is changed, added or deleted? (Required)

Add a link to statement summary tables to help users looking for how to get profiling information in TiDB.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [X] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by dveeden — APPROVED** [review]

**Comment by dveeden** [comment]
We *could* try to come up with a query that mimics `SHOW PROFILES`.

```
mysql> SELECT SLEEP(5);
+----------+
| SLEEP(5) |
+----------+
|        0 |
+----------+
1 row in set (5.01 sec)

mysql> SELECT QUERY_SAMPLE_TEXT,AVG_LATENCY FROM information_schema.statements_summary;
+---------------------------------------------------------------------------------+-------------+
| QUERY_SAMPLE_TEXT                                                               | AVG_LATENCY |
+---------------------------------------------------------------------------------+-------------+
| SELECT SLEEP(5)                                                                 |  5000863698 |
| SELECT QUERY_SAMPLE_TEXT,AVG_LATENCY FROM information_schema.statements_summary |      721116 |
| select @@version_comment limit 1                                                |      499801 |
| SELECT * FROM information_schema.statements_summary                             |     1243165 |
+---------------------------------------------------------------------------------+-------------+
4 rows in set (0.00 sec)
```

File: sql-statements/sql-statement-show-profiles.md [hunk 1]
```diff
@@ -28,4 +28,6 @@ Empty set (0.00 sec)
 
 ## MySQL compatibility
 
-This statement is included only for compatibility with MySQL. Executing `SHOW PROFILES` always returns an empty result.
\ No newline at end of file
+This statement is included only for compatibility with MySQL. Executing `SHOW PROFILES` always returns an empty result.
+
+As an alternative, TiDB provides [statement summary tables](/docs/statement-summary-tables.md) to help understand performance issues.
```
Comment (qiancai) [inline sql-statements/sql-statement-show-profiles.md:33]:
```suggestion
As an alternative, TiDB provides [statement summary tables](/statement-summary-tables.md) to help understand performance issues.
```

**Review by qiancai — APPROVED** [review]

File: sql-statements/sql-statement-show-profiles.md [hunk 2]
```diff
@@ -28,4 +28,6 @@ Empty set (0.00 sec)
 
 ## MySQL compatibility
 
-This statement is included only for compatibility with MySQL. Executing `SHOW PROFILES` always returns an empty result.
\ No newline at end of file
+This statement is included only for compatibility with MySQL. Executing `SHOW PROFILES` always returns an empty result.
+
+As an alternative, TiDB provides [statement summary tables](/statement-summary-tables.md) to help understand performance issues.
```
Comment (qiancai) [inline sql-statements/sql-statement-show-profiles.md:33]:
```suggestion
As an alternative, TiDB provides [statement summary tables](/statement-summary-tables.md) to help understand SQL performance issues.
```

**Pull request merged**